crates/uv/tests/pip_compile.rs

```diff
@@ -6471,23 +6471,23 @@ fn no_strip_markers_multiple_markers() -> Result<()> {
     ----- stdout -----
     # This file was autogenerated by uv via the following command:
     #    uv pip compile --cache-dir [CACHE_DIR] requirements.in --no-strip-markers --python-platform windows
-    attrs==23.2.0 ; python_version > '3.11' or sys_platform == 'win32'
+    attrs==23.2.0 ; sys_platform == 'win32' or python_version > '3.11'
```

Comment:
How is the ordering defined now, is it the ordering of the marker variable flowing down? It looks like in conjunctions and disjunctions the ordering is now different. Not high priority as long as the  ordering is deterministic and robust, but might be easy enough to justify changing

Reply:
It is the ordering of the markers flowing down yeah. In this case the original expression would have been `(python_version <= '3.11' and sys_platform == 'win32') or python_version > '3.11` before simplifying, which is why the order appears reversed despite `python_version` being the highest order variable.

I guess we could sort these the same way we used to?

Reply:
FYI, doing a final sort on simplified markers is preferential for users not seeing a change in equivelent output if something about the resolution itself changes.

Reply:
The resolution is still deterministic, but sorting may result in a more syntactically consistent order.

Reply:
I meant in a scenario where the input changed, e.g. the user changed the order, or added requirements that were previously transative dependencies, or new versions of packages changed the requirements in some similiar sense, but the output was equivelent e.g. same packages, same versions, same markers, but perhaps the marker order could be affected.

Reply:
That should never happen. The decision diagram is canonical for any functionally equivalent marker and the simplification routine is deterministic.